setup/gcp/installer/forseti_installer.py

```diff
@@ -122,15 +121,28 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             bool: Whether or not the deployment was successful
             str: Deployment name
         """
-        deployment_name, return_code = gcloud.create_deployment(
+        deployment_name = gcloud.create_deployment(
             self.project_id,
             self.organization_id,
             deployment_tpl_path,
             self.config.installation_type,
-            self.config.datetimestamp,
+            self.config.timestamp,
             self.config.dry_run)
 
-        deployment_completed = not return_code
+        # Status tracker function, will return true if the deployment is done.
+        status_tracker = (lambda: gcloud.check_deployment_status(
+            deployment_name, constants.DeploymentStatus.DONE))
+        loading_message = ('This may take a few minutes. Waiting '
+                           'for deployment to be completed...')
+        deployment_completed = utils.show_loading(max_loading_time=900,
```

Comment:
This statement would read better as:
`deployment_completed = utils.is_deployment_completed()`

`show_loading` does not reflect what the method is really doing (i.e. check status and return a bool result)

Reply:
Renamed to start_loading instead of show_loading, the method is being used in few places and not only deployment specific.